src/product-service/handlers/getProductsList.test.ts

```diff
@@ -0,0 +1,15 @@
+import { getProductsList } from '../handlers/getProductsList';
+import data from '../data.json';
+
+test('getProductsList returns response', async () => {
+    const response = await getProductsList();
```

Comment:
Please pay attention to AAA pattern in tests: https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80

And lest make a code more readable by adding empty line betwen ACT and ASSERT